endpointmanager/pkg/fetcher/fetcher_test.go

```diff
@@ -52,80 +64,103 @@ func Test_GetEndpointsFromFilepath(t *testing.T) {
+	endpointsCount = len(endpoints.Entries)
+	th.Assert(t, endpointsCount == expectedEndpoints, fmt.Sprintf("Number of endpoints read from CareEvolution file incorrect, got: %d, want: %d.", endpointsCount, expectedEndpoints))
+
+	// test 1Up list
 
+	expectedEndpoints = 472
+	endpoints, _ = GetEndpointsFromFilepath("../../resources/1UpEndpointSources.json", "Lantern", "1Up", "")
+	endpointsCount = len(endpoints.Entries)
+	th.Assert(t, endpointsCount == expectedEndpoints, fmt.Sprintf("Number of endpoints read from 1Up file incorrect, got: %d, want: %d.", endpointsCount, expectedEndpoints))
+
+	// test fhir list
 	expectedEndpoints = 14
-	endpoints, _ = GetEndpointsFromFilepath("../../resources/FHIREndpointSources.json", "FHIR", "")
+	endpoints, _ = GetEndpointsFromFilepath("../../resources/FHIREndpointSources.json", "FHIR", "FHIR", "")
 	endpointsCount = len(endpoints.Entries)
 	th.Assert(t, endpointsCount == expectedEndpoints, fmt.Sprintf("Number of endpoints read from FHIR file incorrect, got: %d, want: %d.", endpointsCount, expectedEndpoints))
 }
 
-func Test_GetListOfEndpointsKnownSource(t *testing.T) {
+func Test_GetListOfEndpointsKnownFormat(t *testing.T) {
 
 	// test cerner list
 
 	cernerListSource := "cerner.com/fhir-endpoints"
-	cernerResult, err := GetListOfEndpointsKnownSource(testCerner, "Cerner", cernerListSource)
+	cernerResult, err := GetListOfEndpointsKnownFormat(testCerner, "Cerner", "Cerner", cernerListSource)
 	th.Assert(t, err == nil, err)
 	th.Assert(t, cernerResult.Entries[0].ListSource == cernerListSource, fmt.Sprintf("The list source should have been %s, it instead returned %s", cernerListSource, cernerResult.Entries[0].ListSource))
 
 	// test epic list
 
-	epicResult, err := GetListOfEndpointsKnownSource(testEpic, "Epic", "")
+	epicResult, err := GetListOfEndpointsKnownFormat(testEpic, "Epic", "Epic", "")
 	th.Assert(t, err == nil, err)
 	th.Assert(t, epicResult.Entries[0].ListSource == "Epic", fmt.Sprintf("The list source should have been Epic, it instead returned %s", epicResult.Entries[0].ListSource))
 
 	// test lantern list
 
-	lanternResult, err := GetListOfEndpointsKnownSource(testLantern, "Lantern", "")
+	lanternResult, err := GetListOfEndpointsKnownFormat(testLantern, "Lantern", "Lantern", "")
 	th.Assert(t, err == nil, err)
 	th.Assert(t, lanternResult.Entries[0].ListSource == "Lantern", fmt.Sprintf("The list source should have been Lantern, it instead returned %s", lanternResult.Entries[0].ListSource))
 
-	// test lantern list
+	// test CareEvolution list
+	careEvolutionResult, err := GetListOfEndpointsKnownFormat(testCareEvolution, "Lantern", "CareEvolution", "")
+	th.Assert(t, err == nil, err)
+	th.Assert(t, careEvolutionResult.Entries[0].ListSource == "CareEvolution", fmt.Sprintf("The list source should have been CareEvolution, it instead returned %s", careEvolutionResult.Entries[0].ListSource))
+
+	// test 1Up list
+	oneUpResult, err := GetListOfEndpointsKnownFormat(test1Up, "Lantern", "1Up", "1Up")
```

Comment:
I think the last string is supposed to be an empty string, to mirror the other tests?